src/core/agent/services/multiSigService.ts

```diff
@@ -1093,6 +1096,219 @@ class MultiSigService extends AgentService {
 
     return { op, exnSaid: exn.ked.d };
   }
+
+  async offerPresentMultisigACDC(
+    multisigSignifyName: string,
+    notificationSaid: string,
+    acdcDetail: any,
+    issuerAidPrefix: string,
```

Comment:
It's not the issuer. It's the other party. From consistenty with KERI lets say `discloseePrefix`